spec/models/journal_entry_spec.rb

```diff
@@ -18,4 +19,31 @@
   describe 'associations' do
     it { should belong_to(:user) }
   end
+  let(:user) { create(:user) }
+
+  describe 'JournalEntry' do
+    it 'creates an attachment' do
+      journal_entry_with_document = FactoryBot.create(:journal_entry, user: user)
+      file_path = Rails.root.join('spec/support/pdf_for_testing.pdf')
+      journal_entry_with_document.attached_document.attach(io: File.open(file_path), filename: 'pdf_for_testing.pdf', content_type: 'application/pdf')
```

Comment:
study about `trait` in rspec factory bot and see if you can add attached documents while creating factory instances.

Reply:
ok thank you i will create a backend story regarding it and study it in the future, but i wont address it in this PR given my Deadline for the application is in 2 days.